gateway-service/src/test/java/org/zowe/apiml/gateway/security/service/zosmf/ZosmfServiceTest.java

```diff
@@ -175,6 +175,26 @@ void givenLoginRequestCredentials_whenAuthenticate_thenAuthenticateWithSuccess()
         assertEquals("jwtToken1", response.getTokens().get(ZosmfService.TokenType.JWT));
     }
 
+    @Test
+    void givenLoginRequestCredentials_whenChangePassword_thenChangePasswordWithSuccess() {
```

Comment:
Same as for the previous Test, could we please first clean the Test class and then add test that already conforms to the new structure we agreed upon?